sympy/geometry/ellipse.py

```diff
@@ -537,6 +537,39 @@ def focus_distance(self):
 
         """
         return Point.distance(self.center, self.foci[0])
+    
```

Comment:
Trailing space means any unnecessary blank spaces, like in here
It does not introduce any bug though, but one reason to remove the spaces is because they can make version control messier, if another person unconsciously modify any blank characters.

You can run `bin/test quality` command from your local git bash to check the line where,
or configure your editor to automatically remove every trailing spaces before saving.